packages/backend/API.md

```diff
@@ -18,18 +18,26 @@ import { Stack } from 'aws-cdk-lib';
 export { a }
 
 // @public
-export type Backend<T extends Record<string, ConstructFactory<ResourceProvider>>> = {
-    createStack: (name: string) => Stack;
-} & {
+export type Backend<T extends Record<string, ConstructFactory<ResourceProvider>>> = BackendHelpers & {
     [K in keyof T]: ReturnType<T[K]['getInstance']>['resources'];
 };
 
+// @public (undocumented)
+export type BackendHelpers = {
+    createStack: (name: string) => Stack;
+};
+
+// @public (undocumented)
+export type BackendInput = Record<string, ConstructFactory<ResourceProvider>> & {
+    [K in keyof BackendHelpers]?: never;
+};

```

Comment:
```suggestion
// @public (undocumented)
export type DefineBackendOptions = Record<string, ConstructFactory<ResourceProvider>> & {
    [K in keyof BackendHelpers]?: never;
};

```

or `DefineBackendOptionsBase`/`DefineBackendProps`/`DefineBackendPropsBase`

to be consistent with how we name inputs in other places.

Reply:
Updated to `DefineBackendProps`